pkg/receiptsmigration/migration_test.go

```diff
@@ -39,7 +39,7 @@ func TestIsMigrated(t *testing.T) {
 	}{
 		{
 			name:         "One plugin and receipts",
-			filesPresent: []string{"store/konfig/konfig.sh", "receipts/present"},
+			filesPresent: []string{"bin/konfig/konfig.sh", "receipts/present"},
```

Comment:
this is not very realistic since it created dirs under bin/
let's just ad /konfig.sh 

also if "konfig" the word isn't relevant, let's say "foo", (ditto below)

but it seems like this has simplified the tests a good deal.

Reply:
Yes, makes much more sense!